src/h3lib/lib/algos.c

```diff
@@ -222,32 +223,32 @@ H3Index h3NeighborRotations(H3Index origin, int dir, int* rotations) {
         } else {
             // generated by hand
             // Current digit -> direction -> new digit
-            const int NEW_DIGIT_II[7][7] = {
+            const Direction NEW_DIGIT_II[7][7] = {
                 {0, 1, 2, 3, 4, 5, 6}, {1, 4, 3, 6, 5, 2, 0},
                 {2, 3, 1, 4, 6, 0, 5}, {3, 6, 4, 5, 0, 1, 2},
                 {4, 5, 6, 0, 2, 3, 1}, {5, 2, 0, 1, 3, 6, 4},
                 {6, 0, 5, 2, 1, 4, 3}};
```

Comment:
Is this generated code? It would have been nice to see this as `enum` members instead of raw integers, but I understand this is too much to ask for all of these values.

Reply:
These tables were written by hand. I'll probably update the PR to make them constants as well, although I'm not sure that'll help readability.